overops-java-bullshifier/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
Since the bullishifer doesn't have a service to test a connection against this file can be deleted.